docker-compose.yml

```diff
@@ -2,35 +2,26 @@ version: '3'
 
 services:
   algod:
-    image: "sdk-harness-algod"
+    image: algorand/algorand-sdk-testing-algod:latest
     container_name: sdk-harness-algod
-    build:
-      context: .
-      dockerfile: ./docker/algod/Dockerfile
     ports:
       - 60001:60001
       - 60000:60000
     networks:
       - sdk-harness
   indexer:
-    image: "sdk-harness-indexer"
+    image: algorand/algorand-sdk-testing-indexer:latest
     container_name: sdk-harness-indexer
-    build:
-      context: .
-      dockerfile: ./docker/indexer/Dockerfile
     ports:
       - 59999:8980
     restart: unless-stopped
     networks:
       - sdk-harness
     environment:
       DATABASE_NAME: dataset1
-
-  indexer-db:
+  postgres:
     image: "postgres"
     container_name: sdk-harness-postgres
-    volumes:
```

Comment:
This is needed, it enables the `POSTGRES_MULTIPLE_DATABASES` environment variable which allows us to initialize multiple databases